common/src/main/java/org/opensearch/ml/common/connector/AbstractConnector.java

```diff
@@ -65,11 +65,18 @@ public abstract class AbstractConnector implements Connector {
     protected User owner;
     @Setter
     protected AccessMode access;
+    @Setter
     protected Instant createdTime;
+    @Setter
     protected Instant lastUpdateTime;
     @Setter
     protected ConnectorClientConfig connectorClientConfig;
 
+    public AbstractConnector() {
+        this.createdTime = Instant.now();
+        this.lastUpdateTime = Instant.now();
```

Comment:
`Instant.now()` may change by a few nanoseconds between these two lines.
```suggestion
        this.lastUpdateTime = this.createdTime;
```

Reply:
Good catch, In my newest commit when connectors get indexed we will place the time and we wont touch the constructor of Abstract connector here is the logic that goes on in `TransportCreateConnectorAction.indexConnector(...)`
 ```
          Instant currentTime = Instant.now();
          connector.setCreatedTime(currentTime);
          connector.setLastUpdateTime(currentTime);
```